demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php

```diff
@@ -390,6 +395,8 @@ function (Statement $statement, mixed $value): array {
                     return [];
                 }
             );
+
+            $configBuilder->numberOfAnonymVotes->readable()->updatable();
```

Comment:
Do we also need the number of non anonym votes?

Reply:
@breierDemos we dont need the number of the non anonymVotes or now, it would be good attribute to have though